src/titiler/xarray/pyproject.toml

```diff
@@ -64,7 +64,16 @@ test = [
     "pytest-cov",
     "pytest-asyncio",
     "httpx",
-    "zarr>=2,<3",
+    "zarr",
+    "h5netcdf",
+    "fsspec",
+]
+test-zarr-v2 = [
+    "pytest",
+    "pytest-cov",
+    "pytest-asyncio",
+    "httpx",
+    "zarr<3.0",
     "h5netcdf",
     "fsspec",
 ]
```

Comment:
This is just for testing. I'm more familiar with hatch than pdm, but if we want more comprehensive tests for both minimum versions and/or upstream main branches I could work on setting that up.